SpatialGDK/Source/SpatialGDK/Public/Interop/SpatialSender.h

```diff
@@ -154,3 +157,12 @@ class SPATIALGDK_API USpatialSender : public UObject
 
 	FUpdatesQueuedUntilAuthority UpdatesQueuedUntilAuthorityMap;
 };
+
+template<typename RPCEndpoint>
+void USpatialSender::SendEndpointReadyUpdate(Worker_EntityId EntityId)
+{
+	RPCEndpoint Endpoint;
+	Endpoint.ready = true;
```

Comment:
I'm not sure about having this use templates to enforce the interface. There's only two types and this isn't something where runtime performance matters, but this will make changing the components mildly harder. I'd rather this is two overloads or polymorphic. With two overloads being my preference here as the underlying components in schema are not polymorphic.

Reply:
I will then change it to two functions instead: SendClientEndpointReadyUpdate and SendServerEndpointReadyUpdate.